mico-core/src/main/java/io/github/ust/mico/core/web/ApplicationController.java

```diff
@@ -102,6 +118,7 @@
         return ResponseEntity.ok(new Resource<>(application, getApplicationLinks(application)));
     }
 
+    // TODO: List of services necessary
```

Comment:
Before the merge into master this should be resolved I think?

Reply:
Obsolete, removed now.